scripts/start-stream-metadata-service.sh

```diff
@@ -1,10 +1,30 @@
 #!/bin/bash
 
+# Initialize variables
+RIVER_ENV=""
+RIVER_ROOT=""
```

Comment:
@tak-hntlabs there was already a lot going on in this file, and it's getting more crowded. it's also polluting the root scripts directory. i suggest making the following changes: 

1. move all of this logic under `packages/stream-metadata/dev.sh`
2. set the "dev" script of stream-metadata/package.json to run `./scripts/dev.sh`
3. remove all references to RIVER_ROOT. whoever used to run `./river/scripts/start-stream-metadata-service.sh` should now run `yarn workspace @river/stream-metadata dev`. this will make it straightforward to run this script regardless of where you're running it from.
4. instead of having `RIVER_ENV` set by a command line option, have it set through an environment variable. Do not initialize it to an empty string at line 4. Just let the caller pass it as an environment variable. That should allow you to remove another 10-15 lines from this.

Reply:
great discussions, @mechanical-turk - simplified.